fusor-ember-cli/app/controllers/rhev-options.js

```diff
@@ -3,12 +3,16 @@ import NeedsDeploymentMixin from "../mixins/needs-deployment-mixin";
 
 export default Ember.Controller.extend(NeedsDeploymentMixin, {
 
-  rhevRootPassword: Ember.computed.alias("deploymentController.model.rhev_root_password"),
-  rhevEngineAdminPassword: Ember.computed.alias("deploymentController.model.rhev_engine_admin_password"),
-  rhevDatabaseName: Ember.computed.alias("deploymentController.model.rhev_database_name"),
-  rhevClusterName: Ember.computed.alias("deploymentController.model.rhev_cluster_name"),
-  rhevCpuType: Ember.computed.alias("deploymentController.model.rhev_cpu_type"),
-  rhevIsSelfHosted: Ember.computed.alias("deploymentController.model.rhev_is_self_hosted"),
+  rhevController: Ember.inject.controller('rhev'),
+
+  rhevRootPassword: Ember.computed.alias("model.rhev_root_password"),
+  rhevEngineAdminPassword: Ember.computed.alias("model.rhev_engine_admin_password"),
+  rhevDatabaseName: Ember.computed.alias("model.rhev_database_name"),
+  rhevClusterName: Ember.computed.alias("model.rhev_cluster_name"),
+  rhevCpuType: Ember.computed.alias("model.rhev_cpu_type"),
+  rhevIsSelfHosted: Ember.computed.alias("model.rhev_is_self_hosted"),
+  confirmRhevRootPassword: Ember.computed.alias("deploymentController.confirmRhevRootPassword"),
+  confirmRhevEngineAdminPassword: Ember.computed.alias("deploymentController.confirmRhevEngineAdminPassword"),
```

Comment:
@cfchase, here we just alias to the value in `deploymentController` which was set by the deployments route setupController.